test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp

```diff
@@ -67,6 +67,50 @@ uint64_t clz64(uint64_t _v)
 	return r;
 }
 
+/// Count trailing zeros for uint32. Following WebAssembly rules, it returns 32 for @a _v being zero.
+/// NOTE: the ctz builtin of the compiler may or may not do this
```

Comment:
is this talking about the yul builtin or the C++ compiler?

Reply:
C++ compiler builtin, just like the clz helpers have this comment.